config/pipelines.yml

```diff
@@ -97,8 +97,8 @@ default: &default
                   type: :model
                   value: tag.id
                 tag_set_name:
-                  type: :string
-                  value: pacbio
+                  type: :model
+                  value: tag.set_name
```

Comment:
and this `tags` possibly

Reply:
Same with this, if it is tags, do you loop through each tags set_name, or is this covered with the nested message you mentioned?